.github/workflows/ci.yml

```diff
@@ -15,19 +15,12 @@ on:
 jobs:
   golang-check:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        go-version: [
-          "1.20",
-          "1.21",
-          "1.22",
-        ]
-
     steps:
       - uses: actions/checkout@v4
+
       - uses: actions/setup-go@v5
         with:
-          go-version: ${{ matrix.go-version }}
+          go-version-file: 'go.mod'
```

Comment:
Does this repo go.mod sync with velero fork?

Reply:
currently not (it is 1.22.0 in default branch, but both will be minor synced after this PR merges)

and after an update PR merges upstream, it will be also 1.22